packages/server/ComputationContainer/Test/IntegrationTest/ReadTripleFromBtsTest.hpp

```diff
@@ -39,6 +39,79 @@ void readTriplesTest(const unsigned int jobIdMax, const unsigned int amount)
 
 TEST(ComputationToBtsTest, ReadTriplesTest10_10000) { readTriplesTest(10, 10000); }
 
+TEST(ComputationToBtsTest, InitTripleStoreTest)
+{
+    const unsigned int jobIdMax = 5;
+    const unsigned int amount = 1000;
+    Config *conf = Config::getInstance();
+    auto cc_to_bts = qmpc::ComputationToBts::Client::getInstance();
+
+    if (conf->party_id == 1)
+    {
+        for (unsigned int jobId = 1; jobId <= jobIdMax; jobId++)
+        {
+            auto triples = cc_to_bts->readTriples<FixedPoint>(jobId, amount);
+        }
+
+        // BTSの初期化
+        cc_to_bts->initTripleStore();
+    }
+
+    // partyで足並みを揃えるため
+    std::vector<Share> fixed_triples;
+    for (unsigned int i = 0; i < amount; i++)
+    {
+        Share a(1);
+        Share b(1);
+        Share c(1);
+        fixed_triples.emplace_back(Share(1));
+        fixed_triples.emplace_back(Share(1));
+        fixed_triples.emplace_back(Share(1));
+    }
+    open(fixed_triples);
+    auto _ = recons(fixed_triples);
```

Comment:
It would be confusing if it were in a test, so it would be better to define something like a Sync() function and just call it.